docs/application/design/TOC.md

```diff
@@ -125,11 +125,7 @@
 ## [Writing style](/application/design/wearable/writing-style.md)
 
 # TV
-## Design Principles
-### [Overview](/application/design/tv/design-principles.md)
-### [Simplicity](/application/design/tv/design-principles/simplicity.md)
-### [Natural Intuitiveness](/application/design/tv/design-principles/natural-intuitiveness.md)
-### [Breathing Space](/application/design/tv/design-principles/breathing-space.md)
+## [Design Principles](/application/design/tv/design-principles.md)
 ## Styles
 ### [Overview](/application/design/tv/styles.md)
```

Comment:
Suggestion : It looks like sub pages of styles are also quite short as of design-principles's. Isn't it better to merge sub pages of styles as done to design-principles?

If you are going to keep current structure, please check `Color swatch` item in design/tv/styles.md file. The link is broken.

Reply:
Good idea.

Reply:
Good idea!